src/sparseml/exporters/transforms/gemm_to_matmulinteger_add_cast_mul.py

```diff
@@ -104,6 +104,8 @@ def transform(self, model: ModelProto) -> ModelProto:
 
             self._do_transform(model, match, attr)
 
+        graph = ONNXGraph(model)
+        graph.sort_nodes_topologically()
```

Comment:
Why is this necessary here?

Reply:
Any transform that adds nodes will require a call to this